app/controllers/sessions_controller.rb

```diff
@@ -115,8 +115,15 @@ def redirect_to_correct_page(user)
       redirect_to transition_info_path
     elsif Settings.rollover && user&.state == "transitioned"
       redirect_to rollover_path
+    elsif user&.state == "rolled_over" && user_associated_with_accredited_body?
+      redirect_to accredited_body_new_features_path
     else
       redirect_to session[:redirect_back_to] || root_path
     end
   end
+
+  def user_associated_with_accredited_body?
```

Comment:
the `user` is actually a `session`. We can only display the `accredited_body_new_features` page if the user is associated with an accredited body. Where is the most appropriate place for this logic?